dev-doc/RegenerateLanguages.md

```diff
@@ -0,0 +1,45 @@
+# Regenerate Languages
+
+This document describes things that need to be done when regenerate the test- and production languages.
+
+1. **xtext-core:** execute the `generateTestLanguages` Gradle task
+```
+./gradlew -PuseJenkinsSnapshots=true generateTestLanguages
```

Comment:
there is also `generateXtextLanguage` but that one uses the bootstrap version.
so executing from Eclipse is more up to date

Reply:
Fixed.